python/lsst/sdm/tools/__init__.py

```diff
@@ -0,0 +1,22 @@
+# This file is part of sdm_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from .version import *
```

Comment:
If you like, you can be one of the first people to stop bothering with the version.py file completely.

```python
from importlib.metadata import PackageNotFoundError, version

try:
    __version__ = version("lsst-sdm-tools")
except PackageNotFoundError:
    # Package not installed or scons not run.
    __version__ = "0.0.0"
```
I think you then edit `SConstruct` to use `versionModuleName=None`.

Which reminds me, I don't see SCons files.

Reply:
What needs to be changed in `pyproject.yaml` to support this, if anything?

Reply:
> I don't see SCons files.

I will add these.

Reply:
in pyproject.toml you can remove the bit about lsst_versions writing to the version.py.